### ISSUE TITLE
Order food certain days

### ISSUE THREAD

**Issue by Floormidabel** [Inter-Actief/amelie#286]
Since some restaurants are closed certain days of the week, it would be nice to be able to allow ordering food from there only if the activity is on a day that the given restaurant is open.

**Comment by Sophietje** [comment]
I think this would not be easy to implement. Either we have to crawl the websites of all the restaurants for their opening days or we need someone (probably a board member) who needs to manually check this once a week or so and possibly update the list. This is a lot of work and I would argue that it would be better to see this as the job of the committee who organises the activity.

**Comment by SebastiaandenBoer** [comment]
I tend to agree with this argument (@Sophietje) as I do not see the long-term viability of manually adding the opening hours or crawling ever-changing websites.

**Comment by Kurocon** [comment]
Crawling the websites is absolutely unwanted, those might change at any time and break random stuff.

We can save the dates in the website, and make it easily changeable. Restaurants don't change their opening days often so it should be relatively easy for the board to keep it up to date (Just change it when you notice it's wrong).

**Comment by Sophietje** [comment]
We want to solve this by adding a "description" field to the restaurant model. So that when someone chooses a restaurant to order from, the description field is shown where the board may have put a warning such as "not open on wednesdays'.

**Comment by supertom01** [comment]
The pull request already exists on the old Inter-Actief/amelie-private repository. But it should be transferred (see internal wiki for instructions)
